['text':' ufuncs appear in pickles with a path in numpy.core._multiarray_umath','line_number':5,'multiline':False]
['text':' and so must import from this namespace without warning or error','line_number':6,'multiline':False]
['text':' TODO: Can remove this after the release.','line_number':31,'multiline':False]
['text':' Only print the message.  This has two reasons (for now!):','line_number':59,'multiline':False]
['text':' 1. Old NumPy replaced the error here making it never actually show','line_number':60,'multiline':False]
['text':'    in practice, thus raising alone would not be helpful.','line_number':61,'multiline':False]
['text':' 2. pybind11 simply reaches into NumPy internals and requires a','line_number':62,'multiline':False]
['text':'    new release that includes the fix. That is missing as of 2023-11.','line_number':63,'multiline':False]
['text':'    But, it "conveniently" ignores the ABI version.','line_number':64,'multiline':False]
